package.json

```diff
@@ -86,11 +95,11 @@
     "@types/react-slick": "^0.23.10",
     "babel-loader": "^8.3.0",
     "husky": "^8.0.0",
+    "postcss": "^8.4.21",
```

Comment:
Are we using this library? If yes, where?

Reply:
Yes it was installed with the gatsby boilerplate.